documentation/catalog/guides/dag_testing.md

```diff
@@ -0,0 +1,49 @@
+# Running DAGs in development
```

Comment:
Do you mind changing this title? "In development" can be a bit ambiguous, because I could imagine that we might be iterating on a DAG that's actually deployed in production. I think making the "local running" part of this explicit would help:
```suggestion
# Running DAGs locally
```